docs/types/conversion.rst

```diff
@@ -59,28 +59,36 @@ At the end of this code snippet, ``x`` will have the value ``0xfffff..fd`` (64 h
 characters), which is -3 in the two's complement representation of 256 bits.
 
 If an integer is explicitly converted to a smaller type, higher-order bits are
-cut off::
+cut off:
+
+.. code-block:: solidity
 
     uint32 a = 0x12345678;
     uint16 b = uint16(a); // b will be 0x5678 now
 
 If an integer is explicitly converted to a larger type, it is padded on the left (i.e., at the higher order end).
-The result of the conversion will compare equal to the original integer::
+The result of the conversion will compare equal to the original integer:
+
+.. code-block:: solidity
 
     uint16 a = 0x1234;
     uint32 b = uint32(a); // b will be 0x00001234 now
     assert(a == b);
 
 Fixed-size bytes types behave differently during conversions. They can be thought of as
 sequences of individual bytes and converting to a smaller type will cut off the
-sequence::
+sequence:
+
+.. code-block:: solidity
 
     bytes2 a = 0x1234;
     bytes1 b = bytes1(a); // b will be 0x12
 
 If a fixed-size bytes type is explicitly converted to a larger type, it is padded on
 the right. Accessing the byte at a fixed index will result in the same value before and
-after the conversion (if the index is still in range)::
+after the conversion (if the index is still in range):
+
+.. code-block:: solidity
```

Comment:
I'm a bit suprised that `::` at the end of a sentence was even valid syntax.

Reply:
Yes I agree with you